.vib/tomcat/goss/vars.yaml

```diff
@@ -0,0 +1,22 @@
+binaries:
+  - gosu
+  - java
+  - render-template
+directories:
+  - mode: "0775"
+    paths:
```

Comment:
I think you are missing `/opt/bitnami/tomcat/logs`

Reply:
Fixed now, thanks for the catch!